packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -161,6 +179,14 @@ export abstract class BaseService extends cdk.Construct
     });
   }
 
+  /**
+   * Associate Service Discovery (Cloud Map) service
+   */
+  public addServiceRegistry(registry: ServiceRegistry) {
```

Comment:
Does this need to be public? It's not like any user is going to call this, are they?